lib/jxl/enc_coeff_order.cc

```diff
@@ -169,7 +150,7 @@ void ComputeCoeffOrder(SpeedTier speed, const ACImage& acs,
       PosAndCount* pos_and_val = mem.get();
       size_t offset = CoeffOrderOffset(ord, c);
       JXL_DASSERT(CoeffOrderOffset(ord, c + 1) - offset == sz);
-      float inv_sqrt_sz = 1.0f / std::sqrt(sz);
+      float inv_sqrt_sz = 0.31f / std::sqrt(sz);
```

Comment:
what's the 0.31?

Reply:
An arbitrary number that works better than 0.30 and 0.32 on the small set I tried it on.

BTW I don't know why counting low-amplitude coeffs also towards the 'zeroes' helps, I was assuming that coeff reordering is to make the num_zeroes AC encoding more effective, but that's probably only part of the story.

Reply:
I'd assume it's because we want the coefficient magnitude to decrease along the order, to make entropy coding easier :)